src/main/java/ua/softserveinc/tc/util/LocalDateUtil.java

```diff
@@ -13,9 +14,11 @@
 import java.util.Date;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
-@Slf4j
 public final class LocalDateUtil {
 
+    @Log
```

Comment:
It is wrong. It doesn't work. This annotation only work with bean but not with sipmle classes.